scripts/homework_scanner.py

```
@@ -0,0 +1,353 @@
+                    ),
+                    body_excerpt=extract_excerpt(body),
+                    matched_pattern=matched,
+                    comment_url=str(review.get("html_url", "")),
+                    source_type="review_body",
+                )
+            )
+
+    return scan
+
+
+def build_issue_body(item: HomeworkItem, owner: str, repo: str) -> str:
```

Ruff (0.15.2)

[warning] Unused function argument: `owner`

(ARG001)